utils/database_test.ts

```diff
@@ -16,33 +16,33 @@ const ltest: Module = {
   star_count: 5,
 };
 
-Deno.test({
-  name: "add, remove, list and count modules in database",
-  async fn() {
-    assertEquals(await database.listModules(10, 1), []);
-    assertEquals(await database.countModules(), 0);
+// Deno.test({
+//   name: "add, remove, list and count modules in database",
+//   async fn() {
+//     assertEquals(await database.listModules(10, 1), []);
+//     assertEquals(await database.countModules(), 0);
 
-    await database.saveModule(ltest);
+//     await database.saveModule(ltest);
 
-    assertEquals(await database.listModules(10, 1), [{
-      name: ltest.name,
-      description: ltest.description,
-      star_count: ltest.star_count,
-      search_score: undefined,
-    }]);
-    assertEquals(await database.countModules(), 1);
-    assertEquals(await database.getModule(ltest.name), ltest);
+//     assertEquals(await database.listModules(10, 1), [{
+//       name: ltest.name,
+//       description: ltest.description,
+//       star_count: ltest.star_count,
+//       search_score: undefined,
+//     }]);
+//     assertEquals(await database.countModules(), 1);
+//     assertEquals(await database.getModule(ltest.name), ltest);
 
-    const ltestWith6Stars = { ...ltest, star_count: 6 };
+//     const ltestWith6Stars = { ...ltest, star_count: 6 };
 
-    await database.saveModule(ltestWith6Stars);
-    assertEquals(await database.countModules(), 1);
-    assertEquals(
-      await database.getModule(ltest.name),
-      ltestWith6Stars,
-    );
-  },
-});
+//     await database.saveModule(ltestWith6Stars);
+//     assertEquals(await database.countModules(), 1);
+//     assertEquals(
+//       await database.getModule(ltest.name),
+//       ltestWith6Stars,
+//     );
+//   },
+// });
```

Comment:
Maybe just use `ignore: true` rather than commenting it out? Or what about just deleting it?

I prefer to delete dead code.

Reply:
good point. i forgot i commented out these tests...